browser/css/calculator.css

```diff
@@ -25,6 +27,10 @@
   margin: 20px;
 }
 
+.calculator:focus {
```

Comment:
oh i see, this is tricky. You're trying to make the actual calculator focusable. Thats a neat trick. I tried this too. I ended up just having a '.calculator-active' className that i moved back and forth when the user clicks on a calculator